# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1707939404 -3600
# Parent  2005a0e2d1a534d358016a3f154ca1ec36ed42eb
Branding: Don't show 'About Rights' and Privacy notice at start. Remove "About" and "Resources" from Account Central.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -14,22 +14,17 @@ pref("general.skins.selectedSkin", "clas
 
 #ifdef XP_MACOSX
 pref("mail.biff.animate_dock_icon", false);
 #endif
 
 pref("mail.rights.version", 0);
 
 // Don't show the about:rights notification in debug or non-official builds.
-#ifdef DEBUG
 pref("mail.rights.override", true);
-#endif
-#ifndef MOZILLA_OFFICIAL
-pref("mail.rights.override", true);
-#endif
 
 // At startup, should we check to see if the installation
 // date is older than some threshold
 pref("app.update.checkInstallTime", true);
 
 // The number of days a binary is permitted to be old without checking is defined in
 // thunderbird-branding.js (app.update.checkInstallTime.days)
 
@@ -134,23 +129,25 @@ pref("app.releaseNotesURL", "https://liv
 // URL for "Learn More" for DataCollection
 pref("toolkit.datacollection.infoURL",
      "https://www.mozilla.org/thunderbird/legal/privacy/#telemetry");
 
 // URL for "Learn More" for Crash Reporter.
 pref("toolkit.crashreporter.infoURL",
      "https://www.mozilla.org/thunderbird/legal/privacy/#crash-reporter");
 
-pref("datareporting.healthreport.uploadEnabled", true); // Required to enable telemetry pings.
+pref("datareporting.healthreport.uploadEnabled", false); // Required to enable telemetry pings, also switch off launcher telemetry in
+                                                         // HKCU\SOFTWARE\Thunderbird\Launcher, see bug 1543807 part 1.
 pref("datareporting.healthreport.infoURL", "https://www.mozilla.org/thunderbird/legal/privacy/#health-report");
 
+// MOZ_DATA_REPORTING should not be set, see mail/moz.configure.
 #ifdef MOZ_DATA_REPORTING
 pref("datareporting.policy.dataSubmissionEnabled", true);
 pref("datareporting.policy.dataSubmissionPolicyAcceptedVersion", 0);
-pref("datareporting.policy.dataSubmissionPolicyBypassNotification", false);
+pref("datareporting.policy.dataSubmissionPolicyBypassNotification", true);
 pref("datareporting.policy.currentPolicyVersion", 2);
 pref("datareporting.policy.firstRunURL", "https://www.mozilla.org/thunderbird/legal/privacy/");
 #endif
 
 // Base URL for web-based support pages.
 pref("app.support.baseURL", "https://www.betterbird.eu/support/");
 
 // Base url for web-based feedback pages.
@@ -1311,17 +1308,17 @@ pref("devtools.inspector.draggable_prope
 
 // Telemetry settings.
 
 // Server to submit telemetry pings to.
 pref("toolkit.telemetry.server", "https://incoming-telemetry.thunderbird.net");
 pref("toolkit.telemetry.server_owner", "Thunderbird");
 
 // Determines if Telemetry pings can be archived locally.
-pref("toolkit.telemetry.archive.enabled", true);
+pref("toolkit.telemetry.archive.enabled", false);
 // Enables sending the shutdown ping when Thunderbird shuts down.
 pref("toolkit.telemetry.shutdownPingSender.enabled", true);
 // Enables sending the shutdown ping using the pingsender from the first session.
 pref("toolkit.telemetry.shutdownPingSender.enabledFirstSession", false);
 // Enables sending a duplicate of the first shutdown ping from the first session.
 pref("toolkit.telemetry.firstShutdownPing.enabled", true);
 // Enables sending the 'new-profile' ping on new profiles.
 pref("toolkit.telemetry.newProfilePing.enabled", true);
diff --git a/mail/themes/shared/mail/accountCentral.css b/mail/themes/shared/mail/accountCentral.css
--- a/mail/themes/shared/mail/accountCentral.css
+++ b/mail/themes/shared/mail/accountCentral.css
@@ -6,17 +6,16 @@
 @import url("chrome://messenger/skin/icons.css");
 
 html {
   height: 100%;
 }
 
 :root {
   --bg-color: #ffffff;
-  --bg-watermark: url("chrome://messenger/skin/images/account-watermark.png");
   --header-bg-color: rgba(0, 0, 0, 0.05);
   --accounts-bg-color: rgba(0, 0, 0, 0.03);
   --text-color: #36385A;
   --title-color: #002275;
   --primary-color: #0a84ff;
   --primary-color-hover: #0060df;
   --btn-color: #36385A;
   --btn-color-hover: #FFFFFF;
@@ -24,17 +23,16 @@ html {
   --btn-bg-hover: #0060df;
   --btn-shadow-hover: rgba(0, 0, 0, 0.3);
   --popup-bg: #EDEDF0;
 }
 
 @media (prefers-color-scheme: dark) {
   :root {
     --bg-color: #2f2f33;
-    --bg-watermark: url("chrome://messenger/skin/images/account-watermark-light.png");
     --header-bg-color: rgba(255, 255, 255, 0.05);
     --accounts-bg-color: rgba(255, 255, 255, 0.03);
     --text-color: #f9f9fa;
     --title-color: #fefefe;
     --primary-color: #0a84ff;
     --primary-color-hover: #0a84ff;
     --btn-color: #FFFFFF;
     --btn-color-hover: #FFFFFF;
@@ -42,17 +40,16 @@ html {
     --btn-bg-hover: #0060df;
     --btn-shadow-hover: rgba(0, 0, 0, 0.3);
     --popup-bg: #474749;
   }
 }
 
 :root[lwt-tree-brighttext] {
   --bg-color: #2f2f33;
-  --bg-watermark: url("chrome://messenger/skin/images/account-watermark-light.png");
   --header-bg-color: rgba(255, 255, 255, 0.05);
   --accounts-bg-color: rgba(255, 255, 255, 0.03);
   --text-color: #f9f9fa;
   --title-color: #fefefe;
   --primary-color: #0a84ff;
   --primary-color-hover: #0a84ff;
   --btn-color: #FFFFFF;
   --btn-color-hover: #FFFFFF;
diff --git a/mailnews/base/content/msgAccountCentral.xhtml b/mailnews/base/content/msgAccountCentral.xhtml
--- a/mailnews/base/content/msgAccountCentral.xhtml
+++ b/mailnews/base/content/msgAccountCentral.xhtml
@@ -235,75 +235,11 @@
             id="importButton"
             type="button"
             data-l10n-id="import-label"
             class="btn-hub btn-inline"
             onclick="top.toImport();"
           ></button>
         </aside>
       </section>
-
-      <section class="account-central-section">
-        <h2 class="section-title" data-l10n-id="about-title"></h2>
-
-        <aside class="row-container">
-          <p data-l10n-id="about-paragraph"></p>
-          <p
-            id="donationParagraph"
-            data-l10n-id="about-paragraph-consider-donation"
-          >
-            <a
-              href="https://give.thunderbird.net/en-US/?utm_source=account_hub_tb_release&amp;utm_medium=referral&amp;utm_content=paragraph_text"
-              class="donation-link"
-              data-l10n-name="donation-link"
-              onclick="openLink(event);"
-              onkeypress="if (event.key == 'Enter') { openLink(event); }"
-              tabindex="0"
-            ></a>
-          </p>
-        </aside>
-      </section>
-
-      <section class="account-central-section">
-        <h2 class="section-title" data-l10n-id="resources-title"></h2>
-
-        <aside class="row-container">
-          <a
-            id="featuresLink"
-            href="https://www.thunderbird.net/en-US/features/"
-            class="resource-link"
-            data-l10n-id="explore-link"
-            onclick="openLink(event);"
-            onkeypress="if (event.key == 'Enter') { openLink(event); }"
-            tabindex="0"
-          ></a>
-          <a
-            id="supportLink"
-            href="https://support.mozilla.org/products/thunderbird"
-            class="resource-link"
-            data-l10n-id="support-link"
-            onclick="openLink(event);"
-            onkeypress="if (event.key == 'Enter') { openLink(event); }"
-            tabindex="0"
-          ></a>
-          <a
-            id="involvedLink"
-            href="https://www.thunderbird.net/en-US/get-involved/"
-            class="resource-link"
-            data-l10n-id="involved-link"
-            onclick="openLink(event);"
-            onkeypress="if (event.key == 'Enter') { openLink(event); }"
-            tabindex="0"
-          ></a>
-          <a
-            id="developerLink"
-            href="https://developer.thunderbird.net/"
-            class="resource-link"
-            data-l10n-id="developer-link"
-            onclick="openLink(event);"
-            onkeypress="if (event.key == 'Enter') { openLink(event); }"
-            tabindex="0"
-          ></a>
-        </aside>
-      </section>
     </main>
   </body>
 </html>
